cypress/e2e/PageObjects/securitySettingsPO.cy.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress" />
+
+
+function securitySettingsHeader(){
+
+    return cy.get('#my-dashboard-heading')
+      }
+
+function breadcrumbs() {
+
+    return cy.get("[class='ds-container'] >nav>ul")
```

Comment:
Getting elements by classes or id is not recommended. Unfortunately, we don't have a way to select the elements needed. I'll send a message to Design System Team about this. 

Here is some [documentation](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)